### ISSUE TITLE
Reader Post Detail VoiceOver: cannot go back after scrolling down then up again

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#11486]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. Go to WPiOS > Reader and click on any article to see the post detail view.
1. Scroll down (using three fingers to swipe up).
1. Scroll back up.
1. Try to go back to the Followed Sites post list.

Result: when using VoiceOver, scrolling down in post detail view makes it so you get stuck on the post and have no way out. ([21s](https://cloudup.com/c6XgKOXqnsR))

<img src="https://user-images.githubusercontent.com/1119271/56321725-859ebb00-6124-11e9-9469-2fc860b58c79.png" width="270" alt="cannot-go-back-from-here-using-voiceover">
<p style="clear:both;"><sup>Tested with WP Internal 12.2.0.20190412 on iPhone 6S iOS 12.2.</sup></p>

<p style="clear:both;"><sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub></p>
